.github/workflows/auto-merge.yml

```diff
@@ -0,0 +1,36 @@
+#
+# Copyright (c) 2019-2020, NVIDIA CORPORATION. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+# A workflow to keep BASE branch up-to-date from HEAD branch
+name: auto-merge HEAD to BASE
+
+on:
+  schedule:
+  - cron: '0 * * * *' # hourly
+
+jobs:
+  auto-merge:
+    runs-on: ubuntu-latest
+    steps:
+    - name: auto-merge job
+      if: ${{ github.ref == 'refs/heads/branch-0.2' }} # make sure this workflow would not be triggered by other refs
+      uses: docker://pxli/automerge:latest # https://gitlab-master.nvidia.com/peixinl/automerge
```

Comment:
is this just a docker image, anything special in it?

Reply:
> is this just a docker image, anything special in it?

The script that does the merge is in that image, therefore most of the work for this feature is in this image.  What's the reasoning for not putting the materials for building the image in this repo somewhere?

Reply:
Agree, the dockerfile and automerge script can be in the public repo (unless I am missing something)?

Reply:
I was thinking that script and dockerfile are supposed to be used as a general tool, so we can setup that workflow in different repos. im totally fine to put it within the repo